woocommerce/woocommerce-android
# Add new login prologue illustration

Closes: #7745


### Description


Adds new illustration to the app on the login prologue screen:

<img width="751" alt="Screenshot 2022-11-23 at 10 35 25" src="https://user-images.githubusercontent.com/266376/203459216-4406f67b-5146-4d15-a966-47ca2e98ed56.png">


### Testing instructions
Start app in logged-out mode and make sure the initial login screen shows the illustration above (including in dark mode/tablet layout/etc).
